src/charset.unit.ts

```diff
@@ -1,10 +1,11 @@
+import { expect } from 'chai'
 import { encode, decode, convert } from './charset'
 
 describe('#charset', function () {
   describe('#encode', function () {
     it('should encode UTF-8 to ArrayBuffer', function () {
       const str = '신'
-      const encoded = new Uint8Array([0xEC, 0x8B, 0xA0])
+      const encoded = new Uint8Array([0xec, 0x8b, 0xa0])
```

Comment:
prettier automatically lower-cases the hex codes. There doesn't seem to be a prettier option around this.
An option would be to use `prettier-eslint` but that comes with other issues (mostly editor integration) and speed of formatting

Reply:
https://github.com/prettier/prettier/issues/5158#issuecomment-425483401